testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -45,10 +46,28 @@
     private static final String TRINO_CATALOG = "iceberg";
     private static final String TEST_SCHEMA_NAME = "default";
 
-    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
-    public void testTrinoReadingSparkData()
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS}, dataProviderClass = TestingIcebergStorageFormat.class, dataProvider = "unsupportedStorageFormats")
+    public void testTrinoWithUnsupportedFileFormat(TestingIcebergStorageFormat storageFormat)
```

Comment:
See comments on other PR. This is overkill since the only unsupported format is Avro.